### ISSUE TITLE
search DNA field doesn't add DNA

### ISSUE THREAD

**Issue by zippy** [holochain/devhub-gui#11]
In trying to create a new release, searching for a DNA and clicking on the "Add to hApp" button fails to add DNA.

![image](https://user-images.githubusercontent.com/3563/164506742-cb28a33b-65dc-4c3e-9df4-5e6e7c465472.png)

This has been intermittent, but once it starts happening I can't ever get it to accept a DNA even though exiting DNA's show up in the drop-down.

**Comment by mjbrisebois** [comment]
Thanks, @zippy 

After investigating I found that it is peculiar behavior in HTML forms that tries to find any `<button>` element in the form and triggers its `click` event instead of triggering the `enter` event on that input.

That flaw was fixed in commit https://github.com/holochain/devhub-gui/commit/12673b70b04bda289eda20625e5647c623007d40 which got included in release v0.5.0.

Are you using version v0.5.0?  If not, I'll have to keep digging.
